config/app.php

```diff
@@ -167,7 +167,7 @@
          */
         Tymon\JWTAuth\Providers\LaravelServiceProvider::class,
         JD\Cloudder\CloudderServiceProvider::class,
-        SimpleSoftwareIO\QrCode\QrCodeServiceProvider::class,
+//        SimpleSoftwareIO\QrCode\QrCodeServiceProvider::class,
```

Comment:
You should not have commented this line. I guess it gave you problem when trying to use `artisan`. un-comment and update your composer.